nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/resources/docs/org.apache.nifi.processors.elasticsearch.JsonQueryElasticsearch/additionalDetails.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>JsonQueryElasticsearch</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+    <p>This processor is intended for use with the ElasticSearch JSON DSL and ElasticSearch 5.X and newer. It is designed
+    to be able to take a query from Kibana and execute it as-is against an ElasticSearch cluster. Like all processors in the
+    "restapi" bundle, it uses the official Elastic client APIs, so it supports leader detection.</p>
+    <p>The query to execute can be provided either in the Query configuration property or in an attribute on a flowfile. In
+    the latter case, the name of the attribute (Expression Language is supported here) must be provided in the Query Attribute
+    property.</p>
+    <p>Additionally, search results and aggregation results can be split up into multiple flowfiles. Aggregation results
+    will only be split at the top level because nested aggregations lose their context (and thus lose their value) if
+    separated from their parent aggregation. The following is an example query that would be accepted:</p>
+    <pre>
+        {
+            "query": {
+                "match": {
+                    "restaurant.keyword": "Local Pizzaz FTW Inc"
```

Comment:
Is this example copy-pasted from somewhere (Elastic, Kibana, etc.)? Just want to make sure we can use it as-is (sometimes they're available under Creative Commons or something), or just to be safe, might want to make our own...

Reply:
Nope, it's all original.

Reply:
Nice, thanks! Will merge shortly